apps/perf-test/tasks/perf-test.js

```diff
@@ -160,43 +217,40 @@ module.exports = async function getPerfRegressions() {
 
   const scenarioList = scenariosArg.length > 0 ? scenariosArg : scenariosAvailable;
 
-  const scenarios = [];
-  scenarioList.forEach(scenario => {
-    if (!scenariosAvailable.includes(scenario)) {
-      throw new Error(`Invalid scenario: ${scenario}.`);
+  /** @type {Scenarios} */
+  const scenarios = {};
+  scenarioList.forEach(scenarioName => {
+    if (!scenariosAvailable.includes(scenarioName)) {
+      throw new Error(`Invalid scenario: ${scenarioName}.`);
     }
     // These lines can be used to check for consistency.
     // Array.from({ length: 20 }, (entry, index) => {
-    scenarios.push({
-      // name: scenario + index,
-      name: scenario,
-      reference: `${urlForMaster}?scenario=${scenario}&iterations=${iterations}`,
-      scenario: `${urlForDeploy}?scenario=${scenario}&iterations=${iterations}`
-    });
-    // });
+    scenarios[scenarioName] = {
+      // scenarios[scenarioName + index] = {
+      baseline: `${urlForMaster}?scenario=${scenarioName}&iterations=${iterations}`,
+      scenario: `${urlForDeploy}?scenario=${scenarioName}&iterations=${iterations}`
+    };
   });
+  // });
 
   console.log(`\nRunning ${iterations} iterations for each of these scenarios: ${scenarioList}\n`);
 
-  if (!fs.existsSync(tempDir)) {
-    console.log(`Making temp directory ${tempDir}...`);
-    fs.mkdirSync(tempDir);
-  }
```

Comment:
Why is it that we don't need to create a `tempDir` anymore if it does not exist?

Reply:
https://github.com/microsoft/flamegrill/pull/14 😄

Reply:
Ahh yes, I remember now!